spec/ParseQuery.Aggregate.spec.js

```diff
@@ -223,9 +223,8 @@ describe('Parse.Query Aggregate testing', () => {
     const obj3 = new TestObject();
     const pipeline = [
       {
-        // TODO: update to new syntax. See [#7339](https://bit.ly/3incnWx)
-        group: {
-          objectId: {
+        $group: {
```

Comment:
Why does the syntax have to be changed and only for this tests, while the other tests still pass with the old syntax?

Reply:
It doesn't have to change, since I was going through postgres tests and applying changes/fixes, I made the change because of the PR linked in the removed comment. That PR didn't make all of the changes to keep code coverage. Either way, the tests pass, but this one checks the updated mongo aggregate syntax against postgres.

Reply:
The fix for this test is in the Postgres Adapter: https://github.com/parse-community/parse-server/blob/d14d5606c386fe3c4f190c18742a8fdf977b2203/src/Adapters/Storage/Postgres/PostgresStorageAdapter.js#L1986-L1987